lib/grape_entity/entity.rb

```diff
@@ -420,6 +420,16 @@ def self.represent(objects, options = {})
       root_element ? { root_element => inner } : inner
     end
 
+    # This method returns the entity's root or collection root node, or its parent's
+    # @param root_type: either :collection_root or just :root
+    def self.root_or_super(root_type)
```

Comment:
Nitpick: I find the method name unnecessarily convoluted. Why don't we just make this `root_element` or `root`?